app/packs/src/apps/mydb/elements/details/samples/analysesTab/SampleDetailsContainersCom.js

```diff
@@ -161,83 +53,118 @@ function RndEdit({
+            general remarks that relate to all analytical data
+          </Tooltip>
+        )}
+      >
+        <Button
+          size="xsm"
+          variant="primary"
+          onClick={() => {setCommentBoxVisible(!commentBoxVisible)}}
+          disabled={disable}
+        >
+          Add comment
+        </Button>
+      </OverlayTrigger>
+    );
+  }
+  
+  const renderCommentBox = (sample, handleCommentTextChange) => {
+    const { container } = sample;
+    return (
+      <Form.Group>
+        <Form.Control
+          as="textarea"
+          style={{ height: '80px' }}
+          value={container.description}
+          onChange={handleCommentTextChange}
+          className="my-3"
+        />
+      </Form.Group>
+    );
+  }
 
   return (
     <div>
-      <p style={{
-        position: 'sticky',
-        top: '0px',
-        zIndex: 1000,
-        backgroundColor: 'white',
-
-      }}
-      >
+      <div className="d-flex justify-content-between align-items-center mb-3">
         {AnalysisModeBtn(mode, toggleMode, isDisabled)}
-        {addButton()}
-        {renderCommentButton(handleCommentButtonClick, false)}
-        {commentBoxVisible ? renderCommentBox(sample, handleCommentTextChange) : null}
-      </p>
-      <PanelGroup
-        id="editable-analysis-list"
-        defaultActiveKey={0}
-        activeKey={activeAnalysis}
-        onSelect={panelOnSelect}
-        accordion
-        style={{
-          position: 'relative',
-          height: '600px',
-          overflowY: 'scroll'
-        }}
-      >
-        {orderContainers.map((container, i) => {
-          const id = container.id || `fake_${i}`;
-          if (container.is_deleted) {
+        <ButtonToolbar className="gap-2">
+          {renderCommentButton()}
+          {addButton()}
+        </ButtonToolbar>
+      </div>
+      {commentBoxVisible && renderCommentBox(sample, handleCommentTextChange)}
+      {mode === 'edit' ? (
+        <Accordion
+          id="editable-analysis-list"
+          onSelect={handleAccordionOpen}
+          activeKey={activeAnalysis}
+        >
+          {orderContainers.map((container, i) => {
+            const id = container.id || `fake_${i}`;
             return (
-              <Panel eventKey={id} key={`${id}CRowEdit`}>
-                <Panel.Heading>{headerDeletedFunc(container)}</Panel.Heading>
-              </Panel>
+              <Accordion.Item eventKey={id} key={`${id}CRowEdit`}>
+                <Accordion.Header>
+                  <AnalysesHeader
+                    sample={sample}
+                    container={container}
+                    mode={mode}
+                    handleUndo={handleUndo}
+                    readOnly={readOnly}
+                    isDisabled={isDisabled}
+                    handleRemove={handleRemove}
+                    handleSubmit={handleSubmit}
+                    toggleAddToReport={toggleAddToReport}
+                  />
+                </Accordion.Header>
```

Comment:
When you are using an `Accordion.Header` and put in the `AnalysesHeader` with all the content and buttons, it will throw a html error.
That's happened because the Accordion.Header generates an H2-Tag with the class 'accordion-header' and inside is a Button. In this Button is the hole content of the `AnalysesHeader`.

I have added a new accordion header at the screen analyses:
https://github.com/ComPlat/chemotion_ELN/blob/update-bootstrap-ui-library/app/packs/src/apps/mydb/elements/details/screens/analysesTab/ScreenDetailsContainers.js

With this new header the hole header is gray and has no white padding. It looks like the analyses on the uncharted server and has a correct html.

Maybe you want to adapt this?

Reply:
Sounds good. I will check it out!